.github/workflows/build.yml

```diff
@@ -1,49 +0,0 @@
-name: Build
-
-on:
-  pull_request:
-  push:
-    branches:
-      - main
-    tags:
-      - v[0-9]+.[0-9]+.[0-9]+
-
-jobs:
-  run:
-    name: Run
-    runs-on: ubuntu-latest
-    steps:
-      - uses: Brightspace/third-party-actions@actions/checkout
-      - uses: Brightspace/third-party-actions@actions/setup-node
-        with:
-          registry-url: 'https://registry.npmjs.org'
-
-      - name: Install Dependencies
-        run: npm install
-
-      - name: Run Linter and Tests
-        run: npm test
-
-      - name: Bundle Host
-        run: npm run bundle-host
-
-      - name: Assume role
-        uses: Brightspace/third-party-actions@aws-actions/configure-aws-credentials
-        if: startsWith(github.ref, 'refs/tags/')
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
-          role-to-assume: "arn:aws:iam::771734770799:role/r+Brightspace+frau-jwt+repo"
-          role-duration-seconds: 3600
-          aws-region: us-east-1
```

Comment:
You'll notice that there's a "bundle" step and a "publish to CDN" step in the old workflows/code that I've removed. They used `browserify` to do the bundling (and `deumdify` to avoid the timebomb) and created a `host.js` and `host-global.js`. From what I can tell from searching around, [host.js isn't used](https://search.d2l.dev/search?project=Brightspace&project=BrightspaceHypermediaComponents&project=BrightspaceUI&project=BrightspaceUILabs&project=lms&full=%22host.js%22+%22frau-jwt%22&defs=&refs=&path=&hist=&type=&xrd=&nn=5&si=full&si=full) and [host-global.js is just used by the LMS](https://search.d2l.dev/search?project=Brightspace&project=BrightspaceHypermediaComponents&project=BrightspaceUI&project=BrightspaceUILabs&project=lms&full=%22host-global.js%22&defs=&refs=&path=&hist=&type=&xrd=&nn=5&si=full&si=full).

My plan is to move the setting of that global [into BSI here](https://github.com/Brightspace/brightspace-integration/blob/main/web-components/ifrau.js) which is already doing iFRA-y things, and update the LMS to not load that file from the CDN.

- [ ] TODO: [remove this from repo-settings](https://github.com/Brightspace/repo-settings/blob/main/repositories/github/Brightspace/frau-jwt.yaml)